Created a new section Deploy Managed Installer. (#9886)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/configure-authorized-apps-deployed-with-a-managed-installer.md

    <RuleCollection Type="ManagedInstaller" EnforcementMode="AuditOnly">
    ```

> An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), Powershell, and Powershell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.

Reviewer: ```suggestion
An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), PowerShell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```

> An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), Powershell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.

Reviewer: ```suggestion
An example of a valid Managed Installer rule collection, using Microsoft Endpoint Config Manager (MEMCM), MEM (Intune), PowerShell, and PowerShell ISE, is shown below. Remove any rules that you do not wish to designate as a Managed Installer.
```

## Enabling managed installer logging events

Refer to [Understanding Application Control Events](event-id-explanations.md#optional-intelligent-security-graph-isg-or-managed-installer-mi-diagnostic-events) for information on enabling optional managed installer diagnostic events.

> ## Deploying the Managed Installer

Reviewer 2: Let's call this "Deploying the Managed Installer rule collection"

Once you've completed configuring your chosen Managed Installer, by specifying which to use in the AppLocker policy, enabling the service enforcement of it, and by enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.

1. Using the following command to deploy the policy.
   ```powershell

>    Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue

Reviewer 2: $policyFile= 
@"
      Raw_AppLocker_Policy_XML
"@

Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue

   ```

> 2. Verify policy deployment

Reviewer 2: ```suggestion
2. Verify Deployment of the Rule set was successful
```

   ```powershell
   Get-AppLockerPolicy -Local
   
   Version RuleCollections RuleCollectionTypes
   ------- --------------- -------------------
      1 {0, 0, 0, 0...} {Appx, Dll, Exe, ManagedInstaller...}
   ```

>     Notice the output shows the ManagedInstaller rule is there.

Reviewer 2: ```suggestion
    Verify the output shows the ManagedInstaller rule set.
```

3. Get the policy XML (optional) using PS:
   ```powershell
   Get-AppLockerPolicy -Effective -Xml -ErrorVariable ev -ErrorAction SilentlyContinue

>    ```

Reviewer 2: At the end, mention that this command will show the raw XML to verify the individual rules (re-word this section)

## Enabling managed installer logging events

Refer to [Understanding Application Control Events](event-id-explanations.md#optional-intelligent-security-graph-isg-or-managed-installer-mi-diagnostic-events) for information on enabling optional managed installer diagnostic events.

> ## Deploying the Managed Installer

Reviewer: ```suggestion
## Deploying the Managed Installer rule collection
```

> Once you've completed configuring your chosen Managed Installer, by specifying which to use in the AppLocker policy, enabling the service enforcement of it, and by enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.

Reviewer: ```suggestion
Once you've completed configuring your chosen Managed Installer by specifying which option to use in the AppLocker policy, enabling the service enforcement of it, and enabling the Managed Installer option in a WDAC policy, you'll need to deploy it.
```

> 1. Using the following command to deploy the policy.

Reviewer: ```suggestion
1. Use the following command to deploy the policy.
```

   ```powershell
   $policyFile=
   @"
   Raw_AppLocker_Policy_XML
   "@
   Set-AppLockerPolicy -XmlPolicy $policyFile -Merge -ErrorAction SilentlyContinue
   ```

> 2. Verify Deployment of the Rule set was successful

Reviewer: ```suggestion
2. Verify deployment of the ruleset was successful.
```

   ```powershell
   Get-AppLockerPolicy -Local
   
   Version RuleCollections RuleCollectionTypes
   ------- --------------- -------------------
   1 {0, 0, 0, 0...} {Appx, Dll, Exe, ManagedInstaller...}
   ```
   Verify the output shows the ManagedInstaller rule set.

> 3. Get the policy XML (optional) using PS:

Reviewer: ```suggestion
3. Get the policy XML (optional) using PowerShell:
```
